streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -155,6 +155,7 @@
     private final StreamsMetricsImpl streamsMetrics;
     private final ProcessorTopology taskTopology;
     private final ProcessorTopology globalTaskTopology;
+    private final Long totalCacheSize;
```

Comment:
Why does this need to be a `Long` instead of a `long`? The numerical value of the variable is only immutable if we use a `long` here.

Reply:
I think that its just what the ide defaulted to. Ill change it.